db/migrate/20230720080631_create_books.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+      t.string :name
```

Comment:
we can also add nullable constraint to each attributes if we don't want them to be nullable

```suggestion
      t.string :name, null: false
```